vtex-apps/structured-data
# Add Search parameter to Search action

**What problem is this solving?**

Structured Data for the search isn't applying the appropriate [search parameter](https://help.vtex.com/tutorial/search-parameters--tutorials_567)

**How should this be manually tested?**

Access this [Workspace](https://searchparameter--hotpointuk.myvtex.com/)

Open developer tools > Elements > search for `potentialAction`
You'll now see the parameter applied to it.
